Modules/ReportBuilder/Handlers/FooterQrCodeBlockHandler.php

```
@@ -0,0 +1,54 @@
+ * Handler for rendering QR code block.
+ *
+ * Expected config structure:
+ * {
+ *   "size": 100,
+ *   "include_url": true,
+ *   "error_correction": "M"
+ * }
+ */
+class FooterQrCodeBlockHandler implements BlockHandlerInterface
+{
+    public function render(BlockDTO $block, Invoice $invoice, Company $company): string
```

PHPMD (2.15.0)

Avoid unused parameters such as '$company'. (undefined)

(UnusedFormalParameter)